src/code/z_debug_display.c

```diff
@@ -16,12 +17,8 @@ static DebugDispObject_DrawFunc sDebugObjectDrawFuncTable[] = {
 };
 
 static DebugDispObjectInfo sDebugObjectInfoTable[] = {
-    { 0, 0x040035F0 }, // Circle
-    { 0, 0x040038F0 }, // Cross
-    { 0, 0x040036F0 }, // Ball
-    { 0, 0x040037F0 }, // Cursor
-    { 1, 0x040039F0 }, // Arrow
-    { 1, 0x04003C90 }, // Camera
+    { 0, gUnknownCircle1Tex }, { 0, gDebugCrossTex }, { 0, gUnknownCircle2Tex },
+    { 0, gDebugTargetTex },    { 1, gDebugArrowDL },  { 1, gDebugCameraDL },
```

Comment:
A few naming suggestions according based on the existing comments:
```suggestion
    { 0, gDebugCircleTex }, { 0, gDebugCrossTex }, { 0, gDebugBallTex },
    { 0, gDebugCursorTex },    { 1, gDebugArrowDL },  { 1, gDebugCameraDL },
```